includes/definitions/splat.yaml

```diff
@@ -0,0 +1,30 @@
+os: splat
+text: 'CheckPoint SecurePlatform'
+type: firewall
+icon: checkpoint
+mib_dir:
+    - checkpoint
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+    - { graph: device_processor, text: 'CPU Usage' }
+    - { graph: device_mempool, text: 'Memory Usage' }
+discovery_modules:
+    ports-stack: 0
+    fdb-table: 0
+    stp: 0
+    wireless: 0
+    vlans: 0
+    bgp-peers: 0
+    ntp: 0
+    services: 0
+    ip6-addresses: 0
```

Comment:
This should be `ipv6-addresses` but does the device not support ipv6?

Reply:
Ipv6 is supported for this platform, but it's not used. Anyway, I'll change it in order to support it